assets/js/components/Work/Tabs/About.jsx

```diff
@@ -1,59 +1,132 @@
+
   return (
     <div className="columns is-centered" data-testid="tab-about-content">
       <div className="column is-three-quarters">
-        <div className="box is-shadowless">
-          <h3 className="small-title">Description</h3>
-          <p>
-            {descriptiveMetadata
-              ? descriptiveMetadata.description
-              : "No description provided"}
-          </p>
-        </div>
-        <div className="box is-shadowless">
-          <h3 className="small-title">Date Created</h3>
-          <p>{formatDate(work.insertedAt)}</p>
-        </div>
-        <div className="box is-shadowless">
-          <h3 className="small-title">Creators</h3>
-          <p className="notification is-warning">
-            Need this exposed in GraphQL
-          </p>
-          <ul>
-            <li>
-              <Link to="/">Creator 1 as a link</Link>
-            </li>
-            <li>
-              <Link to="/">Creator 2 as a link</Link>
-            </li>
-            <li>
-              <Link to="/">Creator 3 as a link</Link>
-            </li>
-          </ul>
-        </div>
-        <div className="box is-shadowless">
-          <h3 className="small-title">Contributors</h3>
-          <p className="notification is-warning">
-            Need this exposed in GraphQL
-          </p>
-          <ul>
-            <li>
-              <Link to="/">Contributor 1 as a link</Link>
-            </li>
-            <li>
-              <Link to="/">Contributor 2 as a link</Link>
-            </li>
-            <li>
-              <Link to="/">Contributor 3 as a link</Link>
-            </li>
-          </ul>
-        </div>
+        <h2 className="title is-size-5">
+          Core Metadata{" "}
+          <a onClick={() => setShowCoreMetadata(!showCoreMetadata)}>
+            <FontAwesomeIcon
+              icon={
+                showCoreMetadata
+                  ? "chevron-circle-down"
+                  : "chevron-circle-right"
+              }
+            />
+          </a>
+        </h2>
+        {showCoreMetadata && (
+          <div className="box">
+            <div className="field">
+              <label className="label">Description</label>
+              <div className="control">
+                {isEditing ? (
+                  <textarea
+                    name="description"
+                    className="textarea"
+                    placeholder="e.g. Hello world"
+                    value={descriptiveMetadata.description}
+                    onChange={e => setDescription(e.target.value)}
+                  />
+                ) : (
+                  <p className="has-text-grey">
+                    {descriptiveMetadata.description ||
+                      "No description provided"}
+                  </p>
+                )}
+              </div>
+            </div>
+            <div className="field">
+              <label className="label">
+                Date Created (Should this be editable?)
```

Comment:
Date created would not be editable.

I had kind of started an issue to determine what needs to happen to make work/file_set editing happen on the backend if you see anything now or later that needs to be added to that. 
https://github.com/nulib/repodev_planning_and_docs/issues/532